common/src/commonTest/kotlin/com/github/mustafaozhan/ccc/common/repo/SettingsRepositoryTest.kt

```diff
@@ -79,12 +80,88 @@ class SettingsRepositoryTest {
     }
 
     @Test
-    fun lastReviewRequest() {
+    fun default_lastReviewRequest() {
         given(settings)
             .function(settings::getLong)
             .whenInvokedWith(matching { it == KEY_LAST_REVIEW_REQUEST }, any())
             .thenReturn(DEFAULT_LAST_REVIEW_REQUEST)
 
         assertEquals(DEFAULT_LAST_REVIEW_REQUEST, repository.lastReviewRequest)
     }
+
+    // setters
+    @Test
+    fun set_firstRun() {
+        given(settings)
+            .function(settings::putBoolean)
+            .whenInvokedWith(matching { it == KEY_FIRST_RUN }, any())
+            .thenReturn(Unit)
+
+        repository.firstRun = true
+
+        verify(settings)
```

Comment:
Since you're verifying the same thing as what you're stubbing, did you know you can use the experimental flag `stubsUnitByDefault` to avoid having to explicitly stub the member before verifying it?

You can read more about it under [Implicit stubbing of functions returning Unit](https://github.com/mockative/mockative#implicit-stubbing-of-functions-returning-unit), and it can be enabled either on the entire project, the test class, or in a specific test function.

Reply:
I think you are speaking in general the ones returning `Unit` right ? Well Since the app is in prod I am trying to avoid `Experimental` stuff, but since it is in Unit test level I think I can use it.